src/Operators/divergences.jl

```diff
@@ -0,0 +1,33 @@
+####
+#### Divergence operators
+####
+
+"""
+    div_cc(i, j, k, grid, u, v)
+
+Calculates the horizontal divergence ∇ₕ·(u, v) of a 2D velocity field (u, v) via
+
+    1/V * [δx_caa(Ax * u) + δy_aca(Ay * v)]
+
+which will end up at the location `cca`.
+"""
+@inline function div_cc(i, j, k, grid, u, v)
```

Comment:
When a function leaves a location unmodified, our convention is to tag that coordinate as `a`. Thus this should be `div_cca`.

Reply:
Also, it might be best to call this `hdiv_caa`, since it is the horizontal divergence.